tools/blueprint/blueprint.css

```
@@ -0,0 +1,222 @@
+@import url(https://www.hlx.page/tools/sidekick/app.css);
```

Let's use from cache:
```suggestion
@import url(https://www.hlx.live/tools/sidekick/app.css);
```